magic/fetcher.py

```diff
@@ -0,0 +1,47 @@
+import csv
+import json
+
+import pkg_resources
+
+from magic import configuration
+from magic.fetcher_internal import (FetchException, fetch, fetch_json, store,
```

Comment:
Here's where the from gets heavy. But I think it's ok. We're a long way from the list of 200 imports at the top of most of the java files I work on in the day job.